.github/workflows/run_linter_powershell.yaml

```diff
@@ -7,13 +6,13 @@ name: Lint Check for PowerShell
 on:
   push:
     paths:
-      - '**.ps1'
-      - '**.psm1'
-      - '**.psd1'
-      - '**.pssc'
-      - '**.psrc'
-      - '**.ps1xml'
-      - '**.cdxml'
+      - "**.ps1"
```

Comment:
What is rationale for changing from single to double quotes?

Reply:
This is a result of using linters on the project.  While there are many ways to represent strings, YAML linters generally consider double-quotes to be safer because it allows escaping characters in an expected manner.  If we don't want to use this check, I can add a new issue to update the YAML linter accordingly.  I don't have dog in this fight personally.